crawlerr/server.py

```
@@ -283,45 +309,55 @@ async def get_server_info(ctx: Context) -> Dict[str, Any]:
-        logger.info("Crawlerr server started successfully ✓")
+        logger.info("[bold green]🎉 Crawlerr server started successfully![/bold green]")
         
     except Exception as e:
-        logger.warning(f"Startup health check failed: {e}")
-        logger.info("Server started but some services may be unavailable")
+        logger.warning(f"[yellow]⚠️  Startup health check failed: {e}[/yellow]")
+        logger.info("[dim]🤷 Server started but some services may be unavailable[/dim]")
 
 
 # CLI entry point
 def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)